packages/auth/amplify_auth_cognito/example/lib/main.dart

```diff
@@ -70,6 +72,15 @@ class _MyAppState extends State<MyApp> {
         ),
       ),
     ),
+    GoRoute(
+      path: '/reset-password',
```

Comment:
Guessing we'll have a separate one for reset password

```suggestion
      path: '/update-password',
```

Reply:
It's in the authenticator.